security.rst

```diff
@@ -2199,30 +2199,55 @@ will happen:
 Thanks to the SensioFrameworkExtraBundle, you can also secure your controller
 using annotations:
 
-.. code-block:: diff
+.. configuration-block::
 
-      // src/Controller/AdminController.php
-      // ...
+    .. code-block:: php-annotations
 
-    + use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+        // src/Controller/AdminController.php
+        // ...
 
-    + /**
-    +  * Require ROLE_ADMIN for *every* controller method in this class.
-    +  *
-    +  * @IsGranted("ROLE_ADMIN")
-    +  */
-      class AdminController extends AbstractController
-      {
-    +     /**
-    +      * Require ROLE_ADMIN for only this controller method.
-    +      *
-    +      * @IsGranted("ROLE_ADMIN")
-    +      */
-          public function adminDashboard(): Response
-          {
-              // ...
-          }
-      }
+        use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+
+        /**
+         * Require ROLE_ADMIN for *every* controller method in this class.
+         *
+         * @IsGranted("ROLE_ADMIN")
+         */
+        class AdminController extends AbstractController
+        {
+            /**
+             * Require ROLE_ADMIN for only this controller method.
+             *
+             * @IsGranted("ROLE_ADMIN")
+             */
+            public function adminDashboard(): Response
+            {
+                // ...
+            }
+        }
+
+    .. code-block:: php-attributes
+
+        // src/Controller/AdminController.php
+        // ...
+
+        use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+
+        /**
+         * Require ROLE_ADMIN for *every* controller method in this class.
```

Comment:
```suggestion
         * Require ROLE_ADMIN for *every* controller actions in this class.
```

what about this? perhaps the controller could have private methods

Reply:
Yes, I've reworded the message a bit.